apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -186,9 +186,7 @@ export default class TeacherHomepage extends Component {
         {isEnglish && showSpecialAnnouncement && (
           <SpecialAnnouncementActionBlock
             hocLaunch={hocLaunch}
-            hasIncompleteApplication={
-              !!sessionStorage['Teacher2021Application']
-            }
+            hasIncompleteApplication={!!sessionStorage['TeacherApplication']}
```

Comment:
Is there any chance of an earlier year's application being loaded at this point?

Reply:
ooh, I guess that could be an issue in future years if they had a partial application in session storage, but this year it wouldn't be an issue since the session storage key name hasn't been used for this before. Also, I think it's unlikely that a user would have a session open a year later with an in-progess application from a previous year - this gets reset if they close the session, right?

Reply:
but we could keep this part year-specific if that feels safer

Reply:
I think the session storage is very [short-lived](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage), so in reality this probably won't be an issue.  But it's worth being aware of.  In fact, the bigger issue might be that we should be using something that lasts longer, but that's probably out of scope here.

If it's easy to parameterize with the current year, that feels like the safest option.  Otherwise, I think getting rid of per-year code changes is worth it in this case.